core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -49,7 +49,8 @@ <h3 class="e2e-test-joyride-title">Preview</h3>
            [ngbTooltip]="'Translations'"
            placement="bottom"
            tabindex="0"
-           aria-label="Exploration Translation Button">
+           aria-label="Exploration Translation Button"
+           (keydown.enter)="!connectedToInternet || editabilityService.isLockedByAdmin() || selectTranslationTab()">
```

Comment:
Hey @Uglycrown what exactly is the `editabilityService.isLockedByAdmin()` here for?

Reply:
@SanjaySajuJacob basically this function is taking boolean value , if current user don't have permission of edit from the admin than that user can't edit the lesson. we are also using this same function if user click on this element.